memfs/memory_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/go-git/go-billy/v5"
 	"github.com/go-git/go-billy/v5/test"
 	"github.com/go-git/go-billy/v5/util"
+	"github.com/stretchr/testify/assert"
 
 	. "gopkg.in/check.v1"
```

Comment:
Would be nice to get rid of `check.v1`

Reply:
That's the plan, I want to do that as a follow-up PR.